tools/kochdocs.nim

```diff
@@ -51,6 +51,13 @@ proc execCleanPath*(cmd: string,
   if execShellCmd(cmd) != 0: quit("FAILURE", errorcode)
   putEnv("PATH", prevPath)
 
+let kochExe* = "koch".absolutePath
```

Comment:
Doesn't look portable to me and even if it is, looks weird, what about ``os.getAppFilename``?

Reply:
done; this was my original idea in previous commit https://github.com/nim-lang/Nim/pull/10183/commits/5bcca5bc75f9ad0264f33887b1f0d685bb3d5bb3#diff-cbed6f667eb622190ddf224587036ea7R438 but I was worried it'd return wrong binary if kochdocs is ever imported by another binary ; the most reliable way that'd avoid all such issues would be to call the API directly without shelling out as I had suggested in original commit (ie `boot()`), but you seemed against it, so that's the best we can do now.